contrib/test_jenkins.sh

```diff
@@ -224,8 +224,9 @@ build_java_docs() {
 	echo " ==== Building java docs ===="
 	if module_load dev/jdk && module_load dev/mvn
 	then
-		pushd ../bindings/java
-		mvn javadoc:javadoc
+		../configure --prefix=$ucx_inst --with-java
+		pushd ../build-test/bindings/java/src/main/native
+		make docs
```

Comment:
why building in source dir and not builddir?

Reply:
This is builddir where `Makefile` is generated

Reply:
so maybe just `make -C bindings/java/src/main/native` ?